target/linux/ramips/image/lzma-loader/src/Makefile

```diff
@@ -54,7 +53,27 @@ LDFLAGS		+= -flto -fwhole-program -Wl,-z,max-page-size=4096
 
 O_FORMAT 	= $(shell $(OBJDUMP) -i | head -2 | grep elf32)
 
-OBJECTS		:= head.o loader.o cache.o board-$(PLATFORM).o printf.o LzmaDecode.o
+OBJECTS		:= head.o loader.o cache.o board.o printf.o LzmaDecode.o
+
+ifeq ($(SUBTARGET),)
+$(error "Please specify a SUBTARGET!")
+endif
+
+ifeq ($(strip $(SUBTARGET)),mt7620)
+CFLAGS	+= -DSOC_MT7620
+endif
+
+ifeq ($(SUBTARGET),mt7621)
+CFLAGS	+= -DSOC_MT7621
+endif
+
+ifeq ($(SUBTARGET),rt305x)
+CFLAGS	+= -DSOC_RT305X
+endif
+
+ifeq ($(SUBTARGET),rt3883)
+CFLAGS	+= -DSOC_RT3883
+endif
```

Comment:
let's use strip in all of them to remain consistent

Reply:
Done!